Prox/Prox/PlaceCarousel/PlaceCarouselViewController.swift

```diff
@@ -193,7 +198,12 @@ class PlaceCarouselViewController: UIViewController {
         // apply the constraints
         NSLayoutConstraint.activate(constraints, translatesAutoresizingMaskIntoConstraints: false)
 
-        toggleLoadingUI(loading: true)
+        toggleLoadingUI(loading: shouldFetchPlaces)
```

Comment:
leftovers from things that I tried to fix the non-interaction of details cards on display. Will remove